### PR TITLE
OpenMP fix to macOS

### PR DESCRIPTION
**Pull request by PhucNgo1711**
Fixes issue #(Enter issue number here)

## Proposed Changes
  -
  -
  -

### CODE REVIEW

**Comment by sarthakpati** [comment]
Is this good to merge?

File: Readme.md [hunk 1]
```diff
@@ -1,56 +1,16 @@
-# CaPTk:  Cancer Imaging Phenomics Toolkit 
+# Binaries
```
Comment (sarthakpati) [inline Readme.md]: This needs to change, please use the original Readme.md. The information about the binaries (applications and Qt) needs to go into a separate file under ```binaries```

File: CMakeLists.txt [hunk 2]
```diff
@@ -427,8 +428,10 @@ IF( NOT BUILD_DOCUMENTATION_ONLY )
   SET( CaPTk_CLIs_Folder "CaPTk_CLIApps")
   SET( CaPTk_Libs_Folder "CaPTk_Libs")
 
-  ADD_SUBDIRECTORY(${PROJECT_SOURCE_DIR}/src)
-  SET(INDIVIDUAL_APPLICATIONS_DIR ${PROJECT_SOURCE_DIR}/src/applications/individualApps)
+  IF (NOT TARGET plugin)
```
Comment (sarthakpati) [inline CMakeLists.txt]: Where is this target coming from? I don't see any references to it.

**Comment by PhucNgo1711** [comment]
> You need to pull from my fork once again.

Yep just pulled the latest stuff

File: CMakeLists.txt (hunk 2 above)
Comment (PhucNgo1711) [reply CMakeLists.txt]: hmm I don't remember making that change. Let me remove that

**Comment by sarthakpati** [comment]
My PR just got merged. I was trying to fix your PR in a separate branch in my fork (which I have merged to my fork's master). Take a look. 

Only this is still unresolved:
- https://github.com/CBICA/CaPTk/pull/598#discussion_r311078523

File: Readme.md (hunk 1 above)
Comment (sarthakpati) [reply Readme.md]: Resolved in my fork.

File: CMakeLists.txt (hunk 2 above)
Comment (PhucNgo1711) [reply CMakeLists.txt]: nvm you just merged my fork

Comment (sarthakpati) [reply CMakeLists.txt]: Can't be; it came in after I pulled from your fork: https://github.com/sarthakpati/CaPTk/commit/818c0b98986fdb75aa67f874186a45bf7f494d73

Comment (sarthakpati) [reply CMakeLists.txt]: Origin of this commit: https://github.com/PhucNgo1711/CaPTk/commit/52dc7430473e6f0c15b495313257fa6d5ed8e511#diff-af3b638bc2a3e6c650974192a53c7291

Comment (sarthakpati) [reply CMakeLists.txt]: And fixed!

File: Readme.md [hunk 3]
```diff
@@ -1,21 +1,14 @@
...
```
Comment (sarthakpati) [inline Readme.md]: You need to completely change this file, BTW. It needs to be the same as https://github.com/CBICA/CaPTk/blob/master/Readme.md

Comment (PhucNgo1711) [reply Readme.md]: Got it
Updated

**Pull request merged**